tools/csv-merger/csv-merger.go

```diff
@@ -222,6 +222,10 @@ func unmarshalStrategySpec(csv *csvv1.ClusterServiceVersion) *csvStrategySpec {
 				Name:  "ROOK_DISABLE_DEVICE_HOTPLUG",
 				Value: "true",
 			},
+		    {
```

Comment:
There seems to be a slight indentation issue here, but it exists in #253 as well, so we'd need to fix it in master first anyway...